src/VisualStudio/Core/Def/Implementation/Experimentation/KeybindingResetDetector.cs

```diff
@@ -174,17 +218,16 @@ private void UpdateStateMachine()
             }
 
             _workspace.Options = options;
-
             if (options.GetOption(KeybindingResetOptions.NeedsReset))
             {
                 ShowGoldBar();
             }
+
+            _isFirstRun = false;
         }
 
         private void ShowGoldBar()
         {
-            AssertIsForeground();
```

Comment:
Why were these all removed?

Reply:
Ah I see, we've moved a bunch of things to the background.

---
In reply to: [245422742](https://github.com/dotnet/roslyn/pull/31918#discussion_r245422742) [](ancestors = 245422742)